system/web/context/RequestContext.cfc

```diff
@@ -740,6 +740,15 @@ component serializable=false accessors="true"{
    		return this;
 	}
 
+    /**
+    * Get the rendered content from a ColdBox integration test
+    *
+    * @return cbox_rendered_content or an empty string
+    */
+    function getRenderedContent(){
```

Comment:
I've noticed you use a way of mixing in functions. I'd love a crash course and then I can implement it here.